src/core/library.c

```diff
@@ -889,6 +889,47 @@ QuicLibrarySetGlobalParam(
 
         break;
 
+    case QUIC_PARAM_GLOBAL_RAW_DATAPATH_PROCS:
```

Comment:
Need to wrap in `{ }` because you declare `uint32_t* RawDataPathProcList` below.